pkg/image/prune/prune.go

```diff
@@ -799,13 +823,14 @@ func (p *pruner) Prune(imagePruner ImageDeleter, streamPruner ImageStreamDeleter
 
 	prunableImageNodes, prunableImageIDs := calculatePrunableImages(p.g, imageNodes)
 	graphWithoutPrunableImages := subgraphWithoutPrunableImages(p.g, prunableImageIDs)
-	prunableLayers := calculatePrunableLayers(graphWithoutPrunableImages)
+	prunableConfigs, prunableLayers := calculatePrunableImageComponents(graphWithoutPrunableImages)
```

Comment:
> I'm not sure why you need to separate lists if both are deleted in exactly the same list.

Originally I wanted to separate deletion of configs from deletion of layers to make it appear nice in the output. But in the end, I don't think it will make the output more readable. There's enough sections being produced already. Adding  a new section `Image Configs` exposes unnecessary details.